site/src/components/AuditLogRow/AuditLogDiff.tsx

```diff
@@ -4,11 +4,15 @@ import { colors } from "theme/colors"
 import { MONOSPACE_FONT_FAMILY } from "theme/constants"
 import { combineClasses } from "util/combineClasses"
 
-const getDiffValue = (value: number | string | boolean) => {
+const getDiffValue = (value: number | string | boolean | null) => {
   if (typeof value === "string") {
     return `"${value}"`
   }
 
+  if (!value) {
+    return `""`
+  }
+
```

Comment:
It's a bit confusing to default all null values to an empty string, when they could really be of any type. I'll try and update the diffs to make sure one of the values is always non-null so you'll be able to know the type.

Reply:
We can return `null` as well. I would avoid rendering custom "empty placeholders" based on value type because what would be an empty placeholder for numbers and booleans? 0 and false? Would not make too much sense to me.

Reply:
I think `null` works :+1:

Reply:
<img width="1576" alt="Screen Shot 2022-09-09 at 13 36 32" src="https://user-images.githubusercontent.com/3165839/189398927-c9285502-674d-4c59-bf1c-d48ebdfb1963.png">